sklearn/calibration.py

```diff
@@ -1041,7 +1041,8 @@ def plot(self, *, ax=None, name=None, ref_line=True, **kwargs):
             created.
 
         name : str, default=None
-            Name for labeling curve.
+            Name for labeling curve. If `None`, the name of the estimator
+            is used.
```

Comment:
This is only true if `from_estimator` is used. If `from_predictions` is used the default is 'Classifier' (with @thomasjpfan's new suggestion). But since this is not public facing API, maybe we can just say if `None` a 'default' value is used...?

Reply:
It is even different. If `name` is `None`, then we use `estimator_name` and if it is also `None` then we don't show anything.